src/content/docs/ko/reference/configuration-reference.mdx

```diff
@@ -527,6 +529,23 @@ Astro에서 생성된 자산 링크의 접두사를 지정합니다. 자산이 
   }
 }
 ```
+**Added in:** `astro@4.5.0`
```

Comment:
Can this be translated, `Added in:`?

Reply:
This can be translated.

However, I think it was a mistake to use `Added in` and not the `<Since v="4.5.0" />` component, so I opened a new PR at #7405.

In this case, should `Added in` be translated?

Reply:
Let's put this PR on hold until those changes are merged then. Thanks for bring it up 😁

Reply:
Thank you!

Reply:
https://github.com/withastro/docs/pull/7405 is an incorrect PR, so this part is translated into Korean.